documentation/howto/tag-and-release-role.md

```diff
@@ -31,35 +31,51 @@ This includes the git tag - the git tag should be **identical** to the version
 number - do not use git tags like `v1.1.0` or `1.1.0-rc` - it must be strictly
 `X.Y.Z`.
 
-## Create Git Tag and Release on Github
+## Update changelogs, Create Git Tag, and Release on Github
 
-Go to the page `https://github.com/linux-system-roles/$ROLENAME/releases/new`
-e.g. https://github.com/linux-system-roles/network/releases/new
+Script [role-make-version-changelog.sh](https://github.com/linux-system-roles/auto-maintenance#role-make-version-changelogsh) is used to create a new version,
+tag, and release for a role. It will guide you through the process.
+It will show you the changes in the role since the last tag, and ask you
+what will be the new semantic version to use for the tag. It will then put
```

Comment:
Just an idea: I guess the semantic version could maybe also be calculated from the changelog, depending on whether it mentions features or not should correspond to the next version.

Reply:
> Just an idea: I guess the semantic version could maybe also be calculated from the changelog, depending on whether it mentions features or not should correspond to the next version.

That's possible.  We already automatically create a new version of the collection, but that version is based on comparing the versions of the roles in that update.   So then to do a release, one would submit a CHANGELOG.md PR with only the 3 sections, and some sort of automation would fill in the version number, the date, etc.